Dockerfile

```diff
@@ -10,19 +10,17 @@ WORKDIR /src
 COPY ./*.sh ./
 RUN shellcheck -e SC1091,SC1090 ./*.sh
 
-FROM mcr.microsoft.com/dotnet/sdk:6.0.100-preview.7-bullseye-slim AS restore
+FROM mcr.microsoft.com/dotnet/sdk:6.0.102-bullseye-slim AS restore
 WORKDIR /src
 COPY ./*.sln ./
 COPY */*.csproj ./
-COPY ./.config/dotnet-tools.json ./.config/
 # Take into account using the same name for the folder and the .csproj and only one folder level
 RUN for file in $(ls *.csproj); do mkdir -p ${file%.*}/ && mv $file ${file%.*}/; done
-RUN dotnet tool restore
 RUN dotnet restore
 
 FROM restore AS build
 COPY . .
-RUN dotnet dotnet-format --check
+RUN dotnet format --verify-no-changes
```

Comment:
It is not a net 6 preview to net 6 upgrade related change. It is out of PR scope but I think that It is an opportunity to include this.